### ISSUE TITLE
Pandora (One?) No Longer Loading

### ISSUE THREAD

**Issue by cizole** [AdAway/AdAway#709]
Hello, so recently I haven't been able to get Pandora to load any more (not sure if having a subscription to Pandora One has anything to do with it). So far I've tried the following:

Disabled AdAway - the app beings working again
Enable AdAway with *.pandora.com added to whitelist - the app still doesn't load (guessing there is some other server that it needs to talk to)

It seems if I turn off the following host list: http://hosts-file.net/ad_servers.txt

Then the app also works, but I hate allowing such a large list of ad-servers just to get Pandora working.

If there is something else I can provide or try that might help I'm all for it.

Also, I can create a separate issue for this (not sure if it may help in cross referencing something), but the Starbucks app is exhibiting the same behavior unless I disable that same host list.

**Comment by 0-kaladin** [comment]
My starbucks app loads fine still.  I don't have pandora so can't speak to that.  However if you narrowed down to the hosts file you can post to their forum http://forum.hosts-file.net/ with the issue and maybe they can narrow down.  what does your starbucks app do?  not open at all?

**Comment by cizole** [comment]
Starbucks just hangs at the logo, the little GPS icon also pop's up. I'm working on trying to narrow down what is interfering with it, but definitely seems to be something being blocked by the hosts file... either: e.crashlytics.com, urbanairship.com (seems like this is an API some devs may use), or something else...still working to narrow that down, but I see entries for those in the blocked hosts and when I re-enable them the app seems to work fine.

I may have spoken too soon on the hosts file. It seems to load 90% of the time now. I turned on the  "Log DNS Requests" option and what I'm noticing if there are different calls made to different hosts. My current theory is that Pandora (as well as Starbucks) is probably trying to hit some kind of tracking, ad, or app usage type URL and doesn't know how to handle not getting a response (not sure if that's by design or just oversight, wouldn't surprise me if they're not punishing people blocking ads).

What I find really annoying though is that I'm not trying to block ad's on Pandora as I have a paid subscription, but again it could be some kind of usage metrics tracking that AdAway is also blocking. I'm going to try to keep experimenting and see if I can narrow down the URL's the apps are relying on.

**Comment by 0-kaladin** [comment]
Enable the AdAway webserver see if that helps you.
